p_sets/2/caesar.rb

```diff
@@ -0,0 +1,42 @@
+# HAIL CAESAR
+# program that encrypts messages using Caesars cipher.
+# program must accept a single command-line argument,k, a non-negative integer.
+# Return 1 if the CLA doesnt meet these criteria
+# Otherwise prompt the user for a string of plaintext and then output ciphertex.
+# non-alphabetical characters should be outputted unchanged.
+# After outputting this ciphertext, your program should exit, returning 0.
+# you may not assume that k will be less than or equal to 26
+#  should work for all non-negative ints less than 2v31 - 26
+require_relative 'encryption'
+class Caesar
+  include Encryption
+
+  def make_key(integer)
```

Comment:
Since this method has no side effects, I probably wouldn't have the `make_` prefix on the name. I can't find a simple, consistent way to phrase it, but generally you don't have to explain that a method is `calculate_`ing or `get_`ting or `make_`ing something - that's implied from the fact that you're calling a method.